Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -559,8 +560,7 @@ public void InitializeColorScheme()
 
         private void QueryTextBox_KeyUp(object sender, KeyEventArgs e)
         {
-            BindingExpression be = QueryTextBox.GetBindingExpression(System.Windows.Controls.TextBox.TextProperty);
-            be.UpdateSource();
+            _viewModel.ChangeQueryText(QueryTextBox.Text);
```

Comment:
Then are we still using Binding? It seems if this looks more like a solution, then we are only bind `OneWay`.

Reply:
Oh wait a moment, you can't use this method, because that will move the cursor to the end. So maybe it will be better to just add a if check for your previous change.

Reply:
Oh, I checked too. so modified different way.